docs/develop/understanding-the-javascript-api-for-office.md

```diff
@@ -161,7 +161,7 @@ For more information, see [Office.initialize Event](https://docs.microsoft.com/j
 
 ## Office JavaScript API object model
 
-Once initialized, the add-in can interact with the host (e.g. Excel, Outlook). The [Office JavaScript API object model](office-javascript-api-object-model.md) page has more details on specific usage patterns. There is also detailed reference documentation for both [shared APIs](https://docs.microsoft.com/office/dev/add-ins/reference/javascript-api-for-office?view=office-js) and specific hosts.
+Once initialized, the add-in can interact with the host (e.g. Excel, Outlook). The [Office JavaScript API object model](office-javascript-api-object-model.md) page has more details on specific usage patterns. There is also detailed reference documentation for both [Common APIs](https://docs.microsoft.com/office/dev/add-ins/reference/javascript-api-for-office?view=office-js) and host-specific APIs.
```

Comment:
Unrelated, but can you strip off the "?view-office-js" from the URL?

Reply:
Sure, I'll make this change globally throughout the doc set.